main.go

```diff
@@ -85,6 +93,10 @@ func main() {
 		esInsecureSkipVerify = kingpin.Flag("es.ssl-skip-verify",
 			"Skip SSL verification when connecting to Elasticsearch.").
 			Default("false").Envar("ES_SSL_SKIP_VERIFY").Bool()
+		esAWS = kingpin.Flag("es.aws", "Enable support for Amazon Elasticsearch Service").
+			Default("false").Envar("ES_AWS").Bool()
+		esAWSRegion = kingpin.Flag("es.aws-region", "Sets the AWS region").
+				Default("").Envar("ES_AWS_REGION").String()
```

Comment:
I have no idea why `go fmt` enforces this indentation... Would fail the CircleCI tests otherwise